### PR TITLE
Switch singularity to ubuntu

### PR DESCRIPTION
**Pull request by Donaim**

### CODE REVIEW

File: Singularity [hunk 1]
```diff
@@ -122,32 +102,23 @@ From: centos:7
     tar -xzf smalt-0.7.6-bin.tar.gz --no-same-owner
     ln -s /opt/smalt-0.7.6-bin/smalt_x86_64 /bin/smalt
 
+    echo ===== Installing Haploflow ===== >/dev/null
+    apt-get install -q -y libboost-all-dev build-essential sudo git ronn cmake
+    cd /opt/
+    git clone https://github.com/hzi-bifo/Haploflow
+    cd Haploflow
+    git checkout 9a5a0ff6c3a0435e723e41f98fe82ec2ad19cf50
+    yes | sh build.sh
```
Comment (donkirkby) [inline Singularity:111]: It looks like Haploflow installs itself into system folders, so you could probably reduce the image size by deleting the git clone.

**Review by donkirkby — APPROVED** [review]
Looks good. One suggestion to try reducing image size.

File: Singularity (hunk 1 above)
Comment (Donaim) [reply Singularity:111]: Ok, I will try!

Comment (Donaim) [reply Singularity:111]: Actually, I just realized that this Haploflow installation wasn't in the original Singularity file. I will move this part to the Haploflow PR.

**Pull request merged**